resources/public/api/conf/1.0/endpoints/amend_benefit.yaml

```diff
@@ -0,0 +1,124 @@
+summary: Amend State Benefit
+description: |
+  This endpoint enables you to amend a previously created customer benefit, if the customer attempts to update
```

Comment:
The endpoint on the spec is slightly different:

`
This endpoint allows a developer to amend a previously created customer benefits. If the customer attempts to update a pre-populated benefit then this operation will fail with a forbidden response code.

A National Insurance number, tax year and benefit ID must be provided.
`